### PR TITLE
changing ... to ellipsis character in json find

### PR DESCRIPTION
**Pull request by RodyLipson**
## References
started fixing #9874 it looks the same in the ui but I checked the inspect element and it did change

### CODE REVIEW

**Review by saulshanabrook — APPROVED** [review]
Thanks! Looks good

**Comment by jasongrout** [comment]
Thanks!

We can merge this as-is, or we can hold off if you want to add more commits of the same fix in other places to this PR. Do you have a preference?

**Review by jasongrout — APPROVED** [review]

**Comment by jasongrout** [comment]
I'll go ahead and merge this in. Feel free to open another PR with more of these fixes. Thanks again!

**Pull request merged**
